tests/functional/test_eclab_parser.py

```diff
@@ -0,0 +1,406 @@
+"""
+The module tests correct parsing of a mpt files generated by the EC-lab with
+use of the Zilien. The dataset with the files exists as a submodule
```

Comment:
maybe add "the submodule 'ixdat-large-test-files'" to prepare for the future, where we might have more than one